### PR TITLE
update machine inactivity limit policy

### PR DESCRIPTION
**Pull request by MaratMussabekov**
Source: https://docs.microsoft.com/en-us/windows/security/threat-protection/security-policy-settings/interactive-logon-machine-inactivity-limit

https://github.com/MicrosoftDocs/windows-itpro-docs/issues/9448

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @JohanFreelancer9,
kindly do the copy editor review for this PR.

Dear @nam31, for your information

Thank you

File: windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md [hunk 1]
```diff
@@ -1045,9 +1045,7 @@ GP Info:
 
 <!--/RegistryMapped-->
 <!--SupportedValues-->
-Valid values:  
-- 0 - disabled 
-- 1 - enabled (session will lock after amount of inactive time exceeds the inactivity limit)
+Valid values: from 0 to 599940, where the value is the amount of inactivity time (in seconds), after which the session will be locked. If it is set to zero (0), the  setting is disabled.
```
Comment (JohanFreelancer9) [inline windows/client-management/mdm/policy-csp-localpoliciessecurityoptions.md:1048]:
```suggestion
Valid values: From 0 to 599940, where the value is the amount of inactivity time (in seconds) after which the session will be locked. If it is set to zero (0), the setting is disabled.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by illfated — APPROVED** [review]
Looks good to me. Thank you for updating this page with the correct value type.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by nam31** [comment]
@Dansimp PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**